File: Data Structures & Algorithms/is-anagram/submission-1.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        
        sort( s.begin() , s.end() );
        sort( t.begin( ) , t.end() );
        for( int i=0; i<s.length(); i++ ) {
            if( s.length() == t.length() ) {
                if( s[i] != t[i] ) {
                    return false;
                }
                
            }
            else {
                return false;
            }
        }
        return true;
    }  
};

File: Data Structures & Algorithms/squares-of-a-sorted-array/submission-1.java
class Solution {
    public int[] sortedSquares(int[] nums) {
        
        int[] arr = new int[nums.length];

        for( int i=0; i<nums.length; i++ ) {
            arr[i] = Math.abs(nums[i]);
        }

        Arrays.sort(arr);

        for( int i=0; i<arr.length; i++ ) {
            nums[i] = arr[i] * arr[i];
        }

        return nums;
    }
}